mollie-payments-for-woocommerce/includes/mollie/wc/helper/settings.php

```diff
@@ -630,4 +618,94 @@ protected function warnWoo3xRequiredForKlarna( $content ) {
 		return $content;
 	}
 
+    /**
+     * Get current locale by WordPress
+     *
+     * Default to self::SETTING_LOCALE_DEFAULT_LANGUAGE
+     *
+     * @return string
+     */
+    protected function getCurrentLocale()
+    {
+        $locale = apply_filters(self::FILTER_WPML_CURRENT_LOCALE, get_locale());
+
+        // Convert known exceptions
+        $locale = $locale === 'nl_NL_formal' ? 'nl_NL' : $locale;
+        $locale = $locale === 'de_DE_formal' ? 'de_DE' : $locale;
+        $locale = $locale === 'no_NO' ? 'nb_NO' : $locale;
+
+        return $this->extractValidLanguageCode([$locale]);
+    }
+
+    /**
+     * Retrieve the browser language
+     *
+     * @return string
+     */
+    protected function browserLanguage()
+    {
+        if (empty($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
```

Comment:
I understand that this is done to save time, especially in the absence of a DI container. But in the future, it would be great to see values like this one be _injected_ into the class, instead of accessing globals from inside the class.

Reply:
For future yes